pub/about.html

```diff
@@ -0,0 +1,24 @@
+<header>
+  Iron Yard Logo
+  </header>
+<h1>About The Iron Yard</h1>
+  <p>The Iron Yard exists to create exceptional value for people and their ideas through code education, 
+  startup accelerators and coworking spaces.</p>
+<h2>We are people-centric.</h2>
+  <p>We put our students, teams, investors and supporters first in all of our decision making. 
+  Our success is entirely dependent on their success.</p>
+<h3>Our challenge is immense.</h3>
+  <p>The world has huge problems that need to be solved, many of which technology is perfectly suited to address. 
+  Companies around the globe are answering this call,
+  but anyone with boots on the ground can feel the acute shortage of talent needed to rise to the occasion. 
+  That gap is widening right now, and over the next decade the shortfall will affect industries, economies,
+  businesses and individuals as they fight to add value in a digital world.</p>
+<h4>The Iron Yard is a solution at that juncture of supply, demand, business and a broken education system.</h4>
+  <p>We are committed to training the highest-quality developers and startups in the world, both in skill and in character.
+  Our mission is to find people who want to pursue the craft and life-long adventure of technology, teach them the tools of the trade, and then release them into the world with the drive and capability to make a difference. 
+  We believe that passionate, talented people who love what they do will create great families, jobs, companies and solutions to the world’s problems.
+  </p>
+<h5>Join us.</h5>
+  <p>We want to accomplish our goal alongside talented, passionate people who share our vision, love what they do, and want to have a whole lot of fun along the way. 
+  Check out our job openings and join the team.</p>
+<footer>©2014 The Iron Yard. All Rights Reserved</footer>
```

Comment:
yea I realized the the headings go in levels of importance but thought they didn't render in the browser but just a way for us to see the order of our webpage but it does effect the text by default as well